zk_toolbox/crates/zk_inception/src/commands/ecosystem/init.rs

```diff
@@ -281,7 +281,7 @@ async fn deploy_ecosystem(
             L1Network::Localhost => {
                 ContractsConfig::get_path_with_base_path(&ecosystem_config.config)
             }
-            L1Network::Sepolia | L1Network::Mainnet => ecosystem_config
```

Comment:
We don't have holesky environment yet, if the Holesky will be chosen this code will fail

Reply:
i think zksync team can deploy a generic ecosystem contracts into holesky for devnets, In currently i think i can just return a error and let user to deploy ecosystem contracts by themself

Reply:
will show:

```
⚙  Command completed: yarn sc build
│  
◇  Provide the path to the ecosystem contracts or keep it empty and you will use ZkSync ecosystem config. For using this config, you need to have governance wallet
│  
│
■  Not found preexisting ecosystem Contracts with chains Holesky
```